### PR TITLE
README copyediting

### PR DESCRIPTION
**Pull request by kethinov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ logger.log('some info')
 logger.warn('a warning')
 logger.error('an error')
 logger.verbose('noisy log only displayed when verbose is enabled')
+logger.log('✅', 'log prepended with a custom emoji or other string')
```
Comment (kethinov) [inline README.md]: note: this is not functionality that currently works, but i'd like to get this in for the initial release. can you implement this and send a commit to my PR, @lannonbr?

the idea behind it is while generally you want to make a custom type if you're going to be using the same prefix for a custom log often, users may sometimes just want to make a one-off log with a one-off prefix, so should be able to do it on the fly

Comment (lannonbr) [reply README.md]: This feature is in the logger

screen shot 2019-01-06 at 3 53 34 pm

Comment (kethinov) [reply README.md]: hah, excellent. not sure why i got an error before, but i tried it again and it does indeed work

**Comment by lannonbr** [comment]
LGTM. Merging

**Pull request merged**